requirements/base.txt

```diff
@@ -68,6 +68,9 @@ boto3==1.9.201
 boto==2.49.0
 moto==2.2.4
 
+# Trino Plugin
+trino==0.305.0
```

Comment:
can you remove Trino as part of the default installation?
The reason is that this won't be a scalable approach to add more query engines.

I know there is no example code for such a use case yet. If you can wait, I will put a PR later this week to remove druid and snowflake dependencies from the repo so you can do something similar.